drivers/spi/spi_ll_stm32.c

```diff
@@ -861,19 +860,26 @@ static void spi_stm32_irq_config_func_##id(struct device *dev)		\
 		.block_count = 2,					\
 	},								\
 	.src_addr_increment = STM32_DMA_CONFIG_##src_dev##_ADDR_INC(	\
-				DMA_CHANNEL_CONFIG(index, dir)),\
+				DMA_CHANNEL_CONFIG(index, dir)),	\
 	.dst_addr_increment = STM32_DMA_CONFIG_##dest_dev##_ADDR_INC(	\
-				DMA_CHANNEL_CONFIG(index, dir)),\
+				DMA_CHANNEL_CONFIG(index, dir)),	\
 	.transfer_complete = false,					\
 	.fifo_threshold = STM32_DMA_FEATURES_FIFO_THRESHOLD(		\
-					DMA_FEATURES(index, dir))	\
-}
+					DMA_FEATURES(index, dir)),	\
+
```

Comment:
missing '}' ?

Reply:
The brackets are now in `SPI_DMA_CHANNEL` macro.
I haven't found other way to achieve what I wanted to do.